app/src/main/java/ca/marshallasch/veil/FragmentViewPost.java

```diff
@@ -59,18 +72,55 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle sa
             postTitle = postObject.getTitle();
             postContent = postObject.getMessage();
             postHash = postObject.getUuid();
+            postDate = Util.timestampToDate(postObject.getTimestamp()).toString();
```

Comment:
This should use `DateFormat.format(Date date)` to make a local independent string. 
See [Date.toString()](https://developer.android.com/reference/java/util/Date#toString())

Reply:
fixed in commit 75b2252